.github/workflows/test_conda_store.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
         working-directory: conda-store
     strategy:
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
+        python-version: ["3.10", "3.11", "3.12"]
```

Comment:
Do we want to include 3.13 here too (and elsewhere or in another PR)?
Best to start testing I would say

Reply:
3.13 isn't possible to test here because [`conda` doesn't yet work with 3.13 ](https://github.com/conda-incubator/conda-store/issues/974) :/. For now I'll leave this be and do this in another PR when we actually have a chance at getting it to pass.